Rollbar/Common/IpAddressUtility.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Rollbar.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    /// <summary>
+    /// A utility class aiding in working with IP addresses.
+    /// </summary>
+    public static class IpAddressUtility
+    {
+        /// <summary>
+        /// Anonymizes the given IP address.
+        /// </summary>
+        /// <param name="exactIpAddress">The specified IP address to anonymize.</param>
+        /// <returns></returns>
+        public static string Anonymize(string exactIpAddress)
+        {
+            if (string.IsNullOrEmpty(exactIpAddress))
+            {
+                return exactIpAddress;
+            }
+
+            string[] components = null;
+
+            // try IPv4 format:
+            const char ipv4CompontsSeparator = '.';
```

Comment:
`Componts` should probably be `Components`

Reply:
another oops. fixed. thank you!